src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -103,6 +103,24 @@ describe('manage nsfs cli account flow', () => {
             assert_account(account_symlink, account_options);
         });
 
+        it('should fail - cli create account invalid access_key (leading space in access_key)', async () => {
+            const { type, name, uid, gid, access_key, secret_key } = defaults;
+            const account_options = { config_root, name, uid, gid, secret_key};
+            const action = ACTIONS.ADD;
+            const command = create_command(type, action, account_options);
+            const res = await exec_manage_cli_add_leading_space_option(command, 'access_key', access_key);
+            expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidArgument.code);
+        });
+
+        it('should fail - cli create account invalid secret_key (leading space in secret_key)', async () => {
+            const { type, name, uid, gid, access_key, secret_key } = defaults;
+            const account_options = { config_root, name, uid, gid, access_key};
```

Comment:
Same here - I would add a short comment that you didn't pass the `secret_key` intentionally.